include/mppic/controller.hpp

```diff
@@ -52,7 +52,9 @@ class Controller : public nav2_core::Controller
   rclcpp::Logger logger_{rclcpp::get_logger("MPPIController")};
   std::shared_ptr<nav2_costmap_2d::Costmap2DROS> costmap_ros_;
   std::shared_ptr<tf2_ros::Buffer> tf_buffer_;
+  std::shared_ptr<rclcpp::Clock> clock_;
```

Comment:
You removed the throttle, you can remove the clock

Reply:
Done